applications/traefik/37.1.2/traefik/cm.yaml

```diff
@@ -103,6 +102,18 @@ data:
               - X-Forwarded-User
               - Impersonate-User
               - Impersonate-Group
+      - # Used by Grafana when impersonation headers cause issues
```

Comment:
was this copied from somewhere? can we document the location where this was taken from if copied?